src/gui/widgets/backupwidget.cpp

```diff
@@ -78,6 +119,7 @@ void BackupWidget::on_createExternalPushButton_clicked()
 
 void BackupWidget::on_restoreExternalPushButton_clicked()
```

Comment:
Same here as above with the backup location, I would advocate for opening `QStandardPaths::HomeLocation`